src/app/Helpers/CheckCategoryTy.php

```diff
@@ -0,0 +1,15 @@
+<?php
+use App\Models\CategoryType;
+    if (!function_exists('checkCategoryTy')) {
+        function checkCategoryTy($id)
+        {
+            $OBJ_Category_Types = CategoryType::all();
+            foreach($OBJ_Category_Types as $OBJ_Category_Type){
+                if ($id == $OBJ_Category_Type->id)
+                {
+                    echo "<option value='".$OBJ_Category_Type->id."' selected>".$OBJ_Category_Type->name."</option>";
+                }
+                    echo "<option value='".$OBJ_Category_Type->id."'>".$OBJ_Category_Type->name."</option>";
+            }
+        }
```

Comment:
Xử lý code này gọn lại nhé.
Tìm hiểu toán tử 3 ngôi để xử lý nhé

Reply:
foreach($OBJ_Category_Types as $OBJ_Category_Type){
$selected = ($id_category_type == $OBJ_Category_Type->id) ? 'selected' : '';
$options .= "<option value='".$OBJ_Category_Type->id."'". $selected.">".$OBJ_Category_Type->name."</option>";
}
return $options;

Hoặc:
$selected = '';
foreach($OBJ_Category_Types as $OBJ_Category_Type){
if ($id_category_type == $OBJ_Category_Type->id) $selected = 'selected';
$options .= "<option value='".$OBJ_Category_Type->id."'". $selected.">".$OBJ_Category_Type->name."</option>"
}
return $options;